drivers/gpu/drm/meson/meson_drv.c

```diff
@@ -1043,7 +1041,7 @@ static void update_scaler(struct drm_crtc *crtc)
 		/* Basic scaler config */
 		osd_reg_write(VPP_OSD_SC_CTRL0,
 			      (1 << 3) /* Enable scaler */ |
-			      (0 << 2) /* Select OSD1 */);
+			      (1 << 2) /* Select OSD1 */);
```

Comment:
I compared the initialization steps against the amlogic driver (and also considering how ORing with 0 doesn't make a lot of sense).